projects/plugins/jetpack-mu-wpcom/src/features/coming-soon/coming-soon.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/**
+ * Coming Soon
+ *
+ * @package A8C\FSE\Coming_soon
+ */
+
+namespace A8C\FSE\Coming_soon;
```

Comment:
Are we intentionally using the same namespace as the ETK plugin? Or do we intend to change that?

Keeping the same namespace might make it easier to deal with any existing dependencies in the wpcom or Atomic code bases. On the other hand, unless it's already using the autoloader to load all the classes (which it looks like it isn't), using the same namespace could be a good way to run into "class is already defined" errors if some manual `require` gets missed when trying to switch things over.

Reply:
> using the same namespace could be a good way to run into "class is already defined" errors if some manual require gets missed when trying to switch things over.

In this case, that sounds like something I'd actually want, each "feature" should only load once and I'd want to know of any other `require` conflicts. I feel like for now at least, migrated stuff can continue using the `A8C\FSE\` namespaces.